src/openfermion/ops/_binary_operator_test.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+from _binary_operator import SymbolicBinary
```

Comment:
please update `__init__.py` in this PR so that you can just write `from openfermion.ops import SymbolicBinary`.